cmd/group-auto-updater/main_test.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"testing"
+
+	coretesting "k8s.io/client-go/testing"
+
+	"k8s.io/apimachinery/pkg/api/equality"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/diff"
+
+	v1 "github.com/openshift/api/user/v1"
+	"github.com/openshift/client-go/user/clientset/versioned/fake"
+)
+
+func TestUpdateGroup(t *testing.T) {
+	peribolosConfig := "test_files/test_peribolos_config.yaml"
```

Comment:
I think the fact that we cannot inline these 10 lines of YAML exposes a smell - IMO the updating logic should not read the filesystem (=take a path to the config as an input) but should  receive a parsed instance and work over that. Then even this test can be simpler and not include hitting a filesystem.

Reply:
Inlining them in the code sounds good.